volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -1761,3 +1764,132 @@ def get_capabilities(self) -> int:
             )
 
         return cap_value & self.get_kernel_cap_full()
+
+
+class timespec64(objects.StructType):
+    def to_datetime(self) -> datetime:
+        """Returns the respective aware datetime"""
+
+        dt = renderers.conversion.unixtime_to_datetime(self.tv_sec + self.tv_nsec / 1e9)
+        return dt
+
+
+class inode(objects.StructType):
+    def is_valid(self) -> bool:
```

Comment:
Would all these be better as properties?  Can't tell, but we kind of had an unofficial rule that additions to a Struct that did stuff started `get_` and otherwise they were made properties.  These looks ideal for making into properties?

Reply:
cool, I will convert those to properties

Reply:
Specifically for the `is_valid()` method, it's not used as property anywhere in the framework:
```shell
$ grep -r -B1 'def is_valid(' | grep property | wc -l
0

$ grep -r -B1 'def is_valid(' | grep -v property | wc -l
104
```

Do you still want me to make it a property?

Reply:
Errr, no, that's ok.  I think that's defined as a part of the interface somewhere so it shouldn't need it.

Reply:
done